src/main/java/uk/gov/hmcts/reform/unspec/model/OrganisationId.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.hmcts.reform.unspec.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Data;
+import lombok.extern.jackson.Jacksonized;
+
+@Data
+@Builder
+@Jacksonized
+public class OrganisationId {
+
+    @JsonProperty("OrganisationID")
+    private String organisationID;
```

Comment:
the only field and class name are same, do we really need this class?

Reply:
This is how the field is structured in CCD,

Reply:
May be we can rename the class property then and leaving the @JsonProperty as it is.